Add References to new C++ Youtube Videos (#6952)
microsoft/vscode-docs: docs/cpp/introvideos-cpp.md

Get started with C++ in Visual Studio Code by watching these introductory videos! These videos are designed to help you set up C++ IntelliSense and build and debug C++ projects in VS Code. After watching these quick tutorials, you'll be able to enjoy VS Code's rich C++ feature set.

## Getting Started with C++ in 5 minutes

> This video shows you how to install the C/C++ Extension and a C++ compiler. It then shows you how to run and debug your code using the play button.

Reviewer: ```suggestion
This video shows you how to install the C/C++ extension and a C++ compiler. You then learn how to run and debug your code by using the **Run and Debug** view.
```
